lib/generators/rolemodel/modals/templates/app/helpers/dynamic_link_helper.rb

```diff
@@ -1,9 +1,9 @@
 module DynamicLinkHelper
   def panel_link_to(name, url, options)
-    link_to(name, url, options.merge(data: { remote: true, panel: true }))
+    link_to(name, url, options.merge(data: { turbo_frame: 'panel' }))
```

Comment:
It's my opinion that the link helpers should be removed entirely.

* I believe they should be removed because they obscure how modals and panels work, making them seem more magical than they really are.
* Links often require application/context specific customization. Which is likely to lead to bloated link helpers as they are adapted to accommodate application specific use-cases.
* I think it's better to use link_to directly and ask developers to add data-turbo-frame='modal' attributes explicitly.

Does anyone else have an opinion on this?

Reply:
We've got more complete versions that we've built into other projects that handle taking a block, etc

Reply:
Feels to me like unnecessary complexity, just to add a single data attribute.  I'm not arguing against helpers, just that they're more project specific.  On NHCS for example, we added this one:

```ruby
  def modal_edit_button(
    resource,
    url: [:edit, resource],
    class: 'btn--small', data: {}
  )
    return unless policy(resource).update?

    icon_button 'pencil', url, class:, data: data.reverse_merge(turbo_frame: 'modal')
  end
```

Reply:
What if we call them `ModelLinkHelper` and `PanelLinkHelper` and support taking a block. Gives us a pattern to build off